services_app/src/androidTest/java/org/opendatakit/utilities/AbstractPermissionsTestCase.java

```diff
@@ -1532,12 +1526,9 @@ private boolean ableToDeleteRow(String rowId, boolean isTableLocked, boolean pri
       return false;
     }
 
-    if ( rowId.startsWith(rowIdFullNull) ) {
```

Comment:
same issue again, please revise all of these in the PR

Reply:
Fixed. Thank you